koala/env.py

```diff
@@ -1,20 +1,21 @@
-import os
-from dotenv import load_dotenv
-
-load_dotenv()
-
-BOT_TOKEN = os.environ['DISCORD_TOKEN']
-BOT_OWNER = os.environ.get('BOT_OWNER')
-
-API_PORT = os.environ.get("API_PORT", 8080)
-
-DB_KEY = os.environ.get('SQLITE_KEY', "2DD29CA851E7B56E4697B0E1F08507293D761A05CE4D1B628663F411A8086D99")
-ENCRYPTED_DB = (not os.name == 'nt') and eval(os.environ.get('ENCRYPTED', "True"))
-
-CONFIG_PATH = os.environ.get("CONFIG_PATH")
-if not CONFIG_PATH:
-    CONFIG_PATH = "/config"
-    if os.name == 'nt':
-        CONFIG_PATH = '.'+CONFIG_PATH
-
-LOGGING_FILE = eval(os.environ.get("LOGGING_FILE", "True"))
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BOT_TOKEN = os.environ['DISCORD_TOKEN']
+BOT_OWNER = [int(os.environ.get('BOT_OWNER'))]
+BOT_OWNER.extend([107153571190935552, 230371277603340289, 142694160887971840, 228541431483072513])

```

Comment:
this is still not quite to the requirements

in .env we expect to see either
```
BOT_OWNER=1234567890
```
or
```
BOT_OWNER=1234567890,2345678901,3456789012
```

therefore we cannot cast this directly to int, we will need to read as a string and parse the input (by using str.split) and creating a list from this parsed output. 

We also do not want to have default values entered in here (as seen on line 8).